### DISCUSSION TITLE
Lightroom

### DISCUSSION THREAD

**Discussion by bobeverywhere** [selfhst/icons#103] [New Icon Request]
Slowly transitioning away from windows and adobe, but still rely heavily on Lightroom for managing photo assets. In the absense of a linux port, web interface becomes the one way to be able to access photo libraries. was looking to add launch icon to my homepage docker service but missing the icon. :-(

**Comment by shollyethan** [comment]
Adobe Lightroom has been added to the collection. I've also added a to-do for myself to add the remaining Adobe CC apps as well. Thanks!
